docs/testing.md

```diff
@@ -0,0 +1,26 @@
+# Testing
+This project has different tests. All tests run in GitHub CI after creating a pull request. Therefore, pull requests can only be merged on successful tests.
```

Comment:
1. Instead of 'project': 'application'?
2. We could omit the reference to 'GitHub' (= implementation detail)?
3. 'on successful tests': 'if all tests have passed' or 'if all tests have run successfully'?